api/gateway/state.go

```diff
@@ -9,17 +9,12 @@ import (
 	"github.com/cosmos/cosmos-sdk/types"
 	"github.com/gorilla/mux"
 
-	"github.com/celestiaorg/celestia-node/blob"
 	"github.com/celestiaorg/celestia-node/state"
 )
 
 const (
-	balanceEndpoint            = "/balance"
-	submitTxEndpoint           = "/submit_tx"
-	submitPFBEndpoint          = "/submit_pfb"
-	queryDelegationEndpoint    = "/query_delegation"
-	queryUnbondingEndpoint     = "/query_unbonding"
-	queryRedelegationsEndpoint = "/query_redelegations"
+	balanceEndpoint  = "/balance"
```

Comment:
oh hmm - @distractedm1nd @vgonkivs can you please help me understand if the `/balance` and `/submit_tx` endpoints are still there or not?

Reply:
We still have both endpoints available through the cmd(rpc client), but for the gateway we've removed `/balance` for self-node in favour of `/balance/addrKey`. Submit_tx is still available for the gateway